### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by novi**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Copyright (c) 2015 Yusuke Ito
+Copyright (c) 2015-17 Yusuke Ito
```
Comment (florianreinhart) [inline LICENSE.md]: It's already 2018 😉

**Comment by florianreinhart** [comment]
LGTM.

Can you merge and release this as new 0.9.0 beta? I'll then test it with my setup.

File: LICENSE.md (hunk 1 above)
Comment (novi) [reply LICENSE.md]: oh, Great. Thanks!

**Comment by novi** [comment]
Thanks for feedbacks. I released this as `0.9.0-beta.2`. We could test this along with our own apps.

File: Sources/MySQL/QueryParameterType.swift [hunk 2]
```diff
@@ -69,12 +69,13 @@ public struct QueryDictionary: QueryParameter {
...
 
 
-protocol QueryArrayType: QueryParameter {
+protocol QueryParameterArrayType: QueryParameter {
     
 }
 
-public struct QueryArray: QueryParameter, QueryArrayType {
-    let arr: [QueryParameter?]
+@available(*, renamed: "QueryArray")
```
Comment (florianreinhart) [inline Sources/MySQL/QueryParameterType.swift:76]: I think this not the correct usage of `renamed`. This annotation should be on the old type and refer to the new name, see: https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Attributes.html

```swift
@available(*, unavailable, renamed: "QueryParameterArray")
typealias QueryArray = QueryParameterArray
```

Comment (novi) [reply Sources/MySQL/QueryParameterType.swift:76]: @florianreinhart Oh, great! Thanks. I will fix that later.

**Pull request merged**